draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3084,49 +3090,86 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
       </t>
 
 
-     <section title="CI/T Objects Versions Capability Object">
+     <section title="CI/T Trigger Endpoint Capability Object" anchor="cit-trigger-endpoints-capability-object">
         <t>
-           The CI/T objects versions capability object is used to indicate support for one or more CI/T objects versions.
-		   Capability type is "FCI.CITObjectsVersion".
-		   For simplicity, it is recommended to minimize the set of supported versions declared by the dCDN. For example, one way to minimize supported versions is to support only objects of version 2.
-           Version 1, as originally defined in <xref target="RFC8007"/>, is the default if this capability is not explicitly declared.
+		The CI/T triger endpoint capability object is used to advertise one or more CI/T interface endpoints 
+		along with CI/T interface versions supported by these endpoints. 
+	   	Capability type is "FCI.CITEndpoints".
+		For simplicity, it is recommended to minimize the set of supported versions declared by the dCDN. 
+		For example, one way to minimize supported versions is to support only objects of version 2.
+           	Version 1, as originally defined in <xref target="RFC8007"/>, is the default if this capability is not explicitly declared.
         </t>
 
-        <t><list style="empty">
-           <t>Property: versions<list style="empty">
+	<t>
+        A CI/T Trigger Endpoints capability object is encoded as an array of JSON objects containing the following
+        name/value pairs:
+	</t>
+
+        <t><list style="empty" hangIndent="3">
+	<t><list style="hanging" hangIndent="3">
 
-                 <t>Description: A list of version numbers.</t>
+		<t hangText="Name: trigger-endpoint-uri">
+		<vspace blankLines="1"/>
+		Description: Trigger endpoint URI 
+		<vspace blankLines="1"/>
+		Value: A URL represented as a JSON string.
+		<vspace blankLines="1"/>
+		Mandatory: Yes.
+		</t>
 
-                 <t>Type: An array of JSON strings</t>
+        	<t hangText="Name: trigger-versions">
+		<vspace blankLines="1"/>
+		Description: A list of CI/T versions supported by the trigger endpoint.
+		<vspace blankLines="1"/>
+		Value: An array of JSON strings.
+		<xref target="loop-detection-and-prevention"/>.
```

Comment:
Looks like another copy-paste error; this line should not be here.

Reply:
Keeping loop detection top of mind!
Fixed